airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/class_types_registry.py

```diff
@@ -59,7 +58,6 @@
     "DefaultErrorHandler": DefaultErrorHandler,
     "DefaultPaginator": DefaultPaginator,
     "DpathExtractor": DpathExtractor,
-    "EmptySchemaLoader": EmptySchemaLoader,
```

Comment:
Removed this entirely since I don't really see a use case where a developer should need to define the default.

Reply:
is it worth adding the default schema loader?

Reply:
That's what I meant that we wouldn't need the default loader defined since we don't expect developers to need to define it themselves.